app/views/state_file/questions/nj_eligibility_health_insurance/edit.html.erb

```diff
@@ -6,20 +6,55 @@
 
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
     <div class="white-group">
-      <%= f.cfa_radio_set(
-            :eligibility_all_members_health_insurance,
-            label_text: t(".label", filing_year: @filing_year),
-            collection: [
-              { value: "yes", label: t("general.affirmative") },
-              { value: "no", label: t("general.negative") },
-            ]
-          ) %>
+      <div class="form-group question-with-follow-up">
+        <div class="question-with-follow-up__question">
+          <%= f.cfa_radio_set(
+                :eligibility_all_members_health_insurance,
+                label_text: t(".label", filing_year: @filing_year),
+                collection: [
+                  { value: "yes", label: t("general.affirmative") },
+                  {
+                    value: "no",
+                    label: t("general.negative"),
+                    input_html: {
+                      "data-follow-up": "#household-members",
+                      "aria-controls": "household-members" 
+                    } 
+                  },
+                ],
+              ) %>
+        </div>
+
+        <div class="form-group question-with-follow-up__follow-up grey-group" id="household-members">
+          <fieldset class="form-group tight-checkboxes">
+            <legend class="form-question not-centered"><%= t('.label_follow_up') %></legend>
+            <div class="input-group--block">
+            <%# TODO: also checkbox for other/self/partner? %>
```

Comment:
[DUST] Prefer not leaving a comment in the code, if this TODO never gets done, the comment just floats there forever. 

Noticed that there's a discussion in the ticket about one potential solution where this gets fixed as a content thing instead of creating a checkbox, so I think that's a reason to remove the comment.